src/containers/App/index.js

```diff
@@ -1,21 +1,51 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import logo from '../../styles/svgs/logo.svg';
 import { useTranslation } from 'react-i18next';
 import LanguageContainer from '../../components/LanguageContainer';
+import appSchema from '../../validations/app';
+import useForm from 'react-hook-form';
+import Validation from '../../components/Validation';
 
 import styles from './index.module.scss';
 
 function App() {
-  const { t } = useTranslation();
+  const [t] = useTranslation();
+  const { register, handleSubmit, errors, triggerValidation } = useForm({
+    validationSchema: appSchema(t),
+    mode: 'onBlur',
+  });
+
+  useEffect(() => {
+    return () => errors && triggerValidation();
+  }, [t]);
```

Comment:
What this does [t]?

Reply:
useEffect takes as argument and array of values, every time one of those values changes useEffect will work.